package.json

```diff
@@ -27,7 +26,7 @@
         "typescript": "^3.7.2"
     },
     "dependencies": {
-        "node-fetch": "^2.6.0",
+        "cross-fetch": "^3.0.4",
```

Comment:
The suggestion was to use `isomorphic-fetch`. But the last commit on the project was on 22 May 2016 (over 3 years ago!) so I consider it unmaintined. 
I have decided to pick `cross-fetch`, which does the same thing, but it's maintined better.